content/en/docs/tasks/administer-cluster/switch-to-evented-pleg.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Switching from polling to CRI event-based updates to container status
+min-kubernetes-server-version: 1.26
+content_type: task
+weight: 90
+---
+
+{{< feature-state for_k8s_version="v1.26" state="beta" >}}
+
+<!-- overview -->
+This page shows how to use `Evented PLEG` (CRI event-based updates to container status) in Kubelet which attempts to lower the resource consumption by avoiding frequent polling for container statuses.
+
+
+## {{% heading "prerequisites" %}}
+
+* This support for this feature is available in Kubernetes {{< skew currentVersion >}};
+  Kubernetes v1.26 as a beta feature, disabled by default.
+* Please make sure the node is [drained](https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/) before proceeding. 
+* Start the `CRI Runtime` with the `Evented PLEG` support. 
+      {{< tabs name="tab_with_code" >}}
+      {{< tab name="Containerd" codelang="bash" >}}
+      Version 1.7+
+      {{< /tab >}}
+      {{< tab name="CRI-O" codelang="bash" >}}
+      v1.26+ and also, start the cri-o daemon with the flag `--enable-pod-events=true` or using a drop in config like,
+            
+      [crio.runtime]
+      enable_pod_events: true
+      
+      {{< /tab >}}
+      {{< /tabs >}}
```

Comment:
At this point, we should have the reader check that their runtime _can_ support events. They don't have to have already reconfigured it (eg CRI-O and `enable_pod_events`).